setup.py

```diff
@@ -218,7 +218,7 @@
     'testing/databaseTest.py',
 ]
 
-modules = []
+modules = ['rmg.py']
```

Comment:
Did you test that this works? I think it should just be `rmg`, but I'm also not sure if you can specify it as both a module and a script.

Reply:
Just to document, we scraped this solution and instead have move the function in rmg.py to util.py where it can be imported.